qiskit/aqua/algorithms/classifiers/qsvm/qsvm.py

```diff
@@ -76,7 +75,7 @@ class QSVM(QuantumAlgorithm):
 
     BATCH_SIZE = 1000
 
-    def __init__(self, feature_map: FeatureMap,
+    def __init__(self, feature_map: Union[QuantumCircuit, FeatureMap],
```

Comment:
Isn't feature_map meaningful - or is the issue that 'circuit' is not in the name?

Reply:
The name feature_map is a very precise name if one has read a paper that introduces it in that way. In general it is a circuit that takes classical data and encodes it in a quantum state, therefore 'data_encoding' or 'data_encoding_circuit' or 'data_circuit' might be a better, more intuitive name. Personally, before knowing the paper the name feature_map confused me quite a bit! Also Jay was pretty keen to no use feature_map.

Reply:
Maybe its better to change that in a follow-up PR, though.

Reply:
data_encoding seems ok, data alone, like data_circuit seems very generic and not very descriptive.

Reply:
Let's figure this out in a follow-up. It's not quite clear to me right now how the varform should be called, since in VQE it is a wavefunction ansatz but in VQC used for classification. Since both derive from `VQAlgorithm` we need to call them the same, don't we?